examples/core/transform-feedback-instanced/app.js

```diff
@@ -138,10 +138,8 @@ const animationLoop = new AnimationLoop({
       id: 'Model-Render',
       vs: DRAW_VS,
       fs: DRAW_FS,
-      geometry: new Geometry({
```

Comment:
I really like this, getting rid of the Geometry here makes the API look much nicer!

Got an idea when reading this, that I am not yet sure about: somehow the `drawMode`/`vertexCount` information could also belong with the TransformFeedback or the VertexArray object.

This is how I think: 
* When you set up those buffers, you usually know best what's in them, what kind of primitives and how many entries.
* Having to remember these params separately so that they can provided here is a bit clunky.

I still like that it can be set/overridden here in Model, but Model could look for default values in TF / VA. What do you think?